build.gradle

```diff
@@ -87,9 +82,29 @@ project('ripc-protocol-tcp') {
 	dependencies {
 		// ripc-core
 		compile project(":ripc-core")
+        compile "org.slf4j:slf4j-api:1.7.6"
+		testCompile 'io.reactivex:rxjava:1.0.8'
```

Comment:
IMO We shouldn't mix layers. We should provide mocks and stick to RS APIs in layers other than composition.